logstage/logstage-core/src/main/scala-3/izumi/logstage/api/logger/AbstractMacroLogIO.scala

```diff
@@ -14,6 +15,58 @@ trait AbstractMacroLogIO[F[_]] { this: AbstractLogIO[F] =>
   transparent inline final def error(inline message: String): F[Unit] = log(Log.Level.Error, message)
   transparent inline final def crit(inline message: String): F[Unit] = log(Log.Level.Crit, message)
 
+  transparent inline final def logMethod[G[x] >: F[x], A](
```

Comment:
Default parameters do work in macros on Scala 3, so overloads may be removed here.

```suggestion
  transparent inline final def logMethod[G[x] >: F[x], A](
    inline level: Level,
    inline logTypes: Boolean = false,
    inline logImplicits: Boolean = false,
  )(inline function: => A
  )(using qp: QuasiIO[G]
  ): G[A] = {
    ${ LogMethodMacro.logMethodIO[A, F, G]('level, 'function, 'this, 'logTypes, 'logImplicits, 'qp) }
  }

  transparent inline final def logMethodF[G[x] >: F[x], A](
    inline level: Level,
    inline logTypes: Boolean = false,
    inline logImplicits: Boolean = false,
  )(inline function: => G[A]
  )(using qp: QuasiPrimitives[G]
  ): G[A] = {
    ${ LogMethodMacro.logMethodIOF[A, F, G]('level, 'function, 'this, 'logTypes, 'logImplicits, 'qp) }
  }
```